### DISCUSSION TITLE
풀이가 있는데 다른 언어로 풀이 Pull Request를 보내도 될까요?

### DISCUSSION THREAD

**Discussion by CoodingPenguin** [tony9402/baekjoon#88] [Question & Answer] [answered]
안녕하세요 이 레포로 알고리즘 스터디를 진행 중인데, 정말 큰 도움이 되고 있습니다!

이에 보답하고자 풀이를 Pull Request로 보내려고 하는데, C++로 작성된 코드가 이미 있더라고요. 근데 저는 Python을 주언어로 써서, 괜찮다면 다른 언어로 된 풀이를 보내도 될까요?

다시 한 번 좋은 레포 감사드립니다!🙏

**Comment by tony9402** [comment, marked as answer]
넵 언어는 Python, Java, C++, JS 모든 언어로 받고 있습니다 !
이미 풀이가 존재하는 언어 제외하고는 자유롭게 PR 날려주시면 됩니다 !

**Reply by CoodingPenguin** [reply]
답변 감사드립니다!
